libraries/Microsoft.Bot.Builder.Dialogs/Control.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Builder.Dialogs
+{
+    /// <summary>
+    /// Base class for controls
+    /// </summary>
+    public abstract class Control : Dialog
+    {
+        private DialogOptions _defaultOptions;
+
+        /// <summary>
+        /// Creates a new Control instance.
+        /// </summary>
+        /// <param name="defaultOptions">(Optional) set of default options that should be passed to controls root dialog. These will be merged with arguments passed in by the caller.</param>
+        public Control(DialogOptions defaultOptions = null)
+        {
+            _defaultOptions = defaultOptions;
```

Comment:
This is sickening. 

Please gen-up a default Dialog Options class and use that if null is passed in.